pipenv/routines/outdated.py

```diff
@@ -60,7 +60,9 @@ def do_outdated(project, pypi_mirror=None, pre=False, clear=False):
                     package_info(package, str(version), str(updated_packages[norm_name]))
                 )
             elif canonicalize_name(package) in outdated_packages:
-                skipped.append(outdated_packages[canonicalize_name(package)])
+                skipped.append(
+                    outdated_packages[canonicalize_name(package)]  # noqa: PLC0206
```

Comment:
I am confused -- why are the noqa markers being added?   Can we not do both?

Reply:
We can specify this just once. The error reporting is misleading, unfortunately.